tests/test_normalize_ast.py

```diff
@@ -81,8 +81,7 @@ def test_visit_ExtSlice():
 
 
 ############ normalize_ast #############
-# TODO: add a test that shows that ast.fix_missing_locations is in use,
-# it is probably better to take tests for ast.fix_missing_locations from python sources.
+
```

Comment:
```suggestion
```

All other similar comments are formatted like this:
```python
<code>


### <comment> ###

<code>
```
so let's stick with it.

Reply:
ok i will stick with it!

Reply:
You can click on the "commit suggestion" button to satisfy the above format.